swan-accpy/Dockerfile

```diff
@@ -35,4 +35,8 @@ RUN pip install --no-deps --no-cache-dir \
     swancustomenvironments==0.0.18
 RUN echo -e '\nc.ServerApp.extra_template_paths = ["/opt/conda/lib/python3.11/site-packages/swancustomenvironments/templates/"]' >> /home/${NB_USER}/.jupyter/jupyter_server_config.py
 
-USER ${NB_UID}
\ No newline at end of file
+# Install UV for faster package installation
+RUN curl -LsSf https://astral.sh/uv/install.sh | XDG_BIN_HOME=/usr/local/bin sh -s -- --no-modify-path && \
```

Comment:
Are there any docs that point to the user of this variable to achieve what you need? What is XDG?

Reply:
If you install the `install.sh` and get the help page, you'll get this
```
uv-installer.sh

The installer for uv 0.5.4

This script detects what platform you're on and fetches an appropriate archive from
https://github.com/astral-sh/uv/releases/download/0.5.4
then unpacks the binaries and installs them to the first of the following locations

    $XDG_BIN_HOME
    $XDG_DATA_HOME/../bin
    $HOME/.local/bin

It will then add that dir to PATH by adding the appropriate line to your shell profiles.

USAGE:
    uv-installer.sh [OPTIONS]

OPTIONS:
    -v, --verbose
            Enable verbose output

    -q, --quiet
            Disable progress output

        --no-modify-path
            Don't configure the PATH environment variable

    -h, --help
            Print help information
```
So, I think is safe to rely on that variable for installing it on the default binary path

Reply:
We then use the `--no-modify-path` for keeping the PATH as it is, since it doesn't need to be updated